__tests__/app.test.js

```diff
@@ -31,6 +31,28 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: return an article object with the correct properties", () => {
+    const article_id = 1;
+
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: article_id,
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+      });
+  });
+});
+
```

Comment:
In this scenario, because we're  requesting a specific article by its ID, we would also need to check the response is a specific article object. I.e. check that an article with article id 1 is being retrieved when we request /api/article/1.
You should add a test to check this, as right now, any article could be being passed with the correct structure.